angular.json

```diff
@@ -497,10 +505,6 @@
               {
                 "command": "nx build-package eslint-plugin"
               },
-              // uncomment when there are eslint-plugin migrations
-              // {
-              //   "command": "yarn tsc -p modules/eslint-plugin/tsconfig.schematics.json"
-              // },
```

Comment:
this change should be reverted I guess 👀

Reply:
I think we mentioned this already, what about leaving it out of the file and re-add it if we need it. We already have it setup for the other packages, so we can use that as a reference.